sig/datadog/appsec/contrib/auto_instrument.rbs

```diff
@@ -0,0 +1,11 @@
+module Datadog
+  module AppSec
+    module Contrib
+      # Auto-instrumentation for security integrations
+      # TODO: this implementation is trivial, check for shareable code with tracer
```

Comment:
Leftover comment ;)

Should we maybe have it part of the CI check that there are no comments here, or is that a bit too agressive?

Reply:
(There's a few other .rbs files with comments as well)

Reply:
I think we do want comments in `.rbs`, only related to typing, e.g to explain why this type or alias is that way, or TODOs, or... you get the point!

Reply:
(Good catch though)

Reply:
Good point. I guess we can keep an eye out for "oops this comment should not be in" during reviews and at some point it should get pretty rare because we'll not be creating new `.rbs` files very often.